sycl/test-e2e/Graph/RecordReplay/prefetch.cpp

```diff
@@ -0,0 +1,9 @@
+// REQUIRES: cuda || level_zero, gpu
+// RUN: %{build} -o %t.out
+// RUN: env SYCL_PI_TRACE=2 %{run} %t.out
+
+// CHECK: piextCommandBufferPrefetchUSM
+
```

Comment:
normally we also do
```
// Extra run to check for leaks in Level Zero using ZE_DEBUG
// RUN: %if ext_oneapi_level_zero %{env ZE_DEBUG=4 %{run} %t.out 2>&1 | FileCheck %s %}
//
// CHECK-NOT: LEAK
```

Was this intentionally left out in favor of `SYCL_PI_TRACE`?

Reply:
Since Prefetch and Advise operations do not impact output results but only performance, I use the PI_TRACE to verify that the backend functions are called.
The comments explaining this were probably not in the appropriate files, I thus moved them to the files where the environment variable is set.